Sources/hostmgr/main.swift

```diff
@@ -23,23 +23,24 @@ struct HostMgr: ParsableCommand {
     var version: Bool = false
 
     func run() throws {
+        if version {
+            print(appVersion)
+            return
+        }
```

Comment:
Nitpick. Given we want this to return, what do you think of using `guard` so the compiler will help us?

```suggestion
        guard version == false else {
            print(appVersion)
            return
        }
```